src/main/java/com/twentyonec/ItemsLogger/ItemsLogger.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Cheran (21C)
+ * 
+ * Copyright (c) 2020 21C, Inc. All Rights Reserved.
+ */
+package com.twentyonec.ItemsLogger;
+
+import java.util.logging.Level;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.java.JavaPlugin;
+
+import com.twentyonec.ItemsLogger.commands.Commands;
+import com.twentyonec.ItemsLogger.commands.OpenItemLog;
+import com.twentyonec.ItemsLogger.listeners.DeathSave;
+import com.twentyonec.ItemsLogger.utils.Config;
+import com.twentyonec.ItemsLogger.utils.Storage;
+
+/**
+ * Plugin main class
+ * 
+ * @version 1.0.1 22 October 2020
+ * @author Cheran (21C)
+ */
+public class ItemsLogger extends JavaPlugin {
+
+	static ItemsLogger plugin = null;
+	Config config = null;
+	Storage storage = null;
+
+	@Override
+	public void onEnable() {
+		plugin = this;
+		saveDefaultConfig();
+		config = new Config(getConfig());
+		storage = Storage.getStorage(this);
+		storage.setUpTable();
```

Comment:
Get into the habit of referring to the owner of data. It makes explicitly clear which variables are `static`, which belong to the class, and which are scoped to the containing method.
```suggestion
		ItemsLogger.plugin = this;
		this.saveDefaultConfig();
		this.config = new Config(getConfig());
		this.storage = Storage.getStorage(this);
		this.storage.setUpTable();
```
This goes for everywhere where you've forgotten `this`.